### PR TITLE
arcade blog updates

### PR DESCRIPTION
**Pull request by mmoskal**

### CODE REVIEW

File: docs/blog/arcade/01-18-2019.md [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
-able to buy or build your own MakeCode Arcade compatible game systems. 
+for the web and for dedicated hardware.
+As usual with MakeCode, you can 
+either use blocks or JavaScript to build your program in your browser.
+There's nothing to install - the games will just work in the browser!
+For even more fun, you'll be
+able to build your own Arcade-compatible game systems or buy them from
+our hardware partners.
+But just to make it clear - this time around hardware is optional, and you
+can get started playing or teaching using only a computer with a web-browser.
```
Comment (mmoskal) [inline docs/blog/arcade/01-18-2019.md]: Emphasized that hardware is optional

**Comment by pelikhan** [comment]
We need animated gifs of kick ass games. Use particles too.

File: docs/blog/arcade/01-18-2019.md [hunk 2]
```diff
@@ -31,33 +48,85 @@ along with a gallery of specially designed sprites for your use:
...
 ### Game APIs
 
 MakeCode Arcade comes with a set of game APIs to make it simple to get
 started programming your game, including:
 * **Sprite** physics, effects, collisions, animations and lifetime management;
 * **Controllers**, both single-player and multi-player;
-* **Music**, XYZ
-* **Game** updates, prompots and dialogs;
+* **Music**, 4-channel sound, different waveforms and envelopes; no samples;
+* **Game** updates, prompts and dialogs;
```
Comment (pelikhan) [inline docs/blog/arcade/01-18-2019.md]: Tile map

**Comment by Jaqster** [comment]
Looks excellent!   Should we ask for feedback/bugs?

**Comment by pelikhan** [comment]
Script editor in TypeScript?

**Comment by mmoskal** [comment]
it already mentions that, but maybe a screenshot would be nice

**Comment by mmoskal** [comment]
@Jaqster 
feedback/questions -> forum?
bugs -> github (or forum if no account)?

**Comment by Jaqster** [comment]
I had set up a couple shortcut links - http://aka.ms/arcadechat, http://aka.ms/arcadebugs

**Comment by Jaqster** [comment]
I thought the forum was just for sharing games...?

**Pull request merged**